commons/src/main/java/com/powsybl/openrao/commons/NumberRounding.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
```

Comment:
General comment : quid des logs ?

Reply:
out of scope since values are logged with all decimals anyway

Reply:
but I will update for the logs though :ok_hand: